app-emulation/fex-rootfs-gentoo/fex-rootfs-gentoo-20240909.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd linux-info
+
+DESCRIPTION="A x86 system image used for FEX"
+HOMEPAGE="https://github.com/WhatAmISupposedToPutHere/fex-rootfs"
+
+SRC_URI="
+	https://github.com/WhatAmISupposedToPutHere/fex-rootfs/releases/download/${PV}/fex-rootfs.sqfs -> ${P}-root.sqfs
+	https://github.com/WhatAmISupposedToPutHere/fex-rootfs/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz
+"
+
+S="${WORKDIR}/fex-rootfs-${PV}"
+
+LICENSE="metapackage MIT"
+SLOT="0"
+KEYWORDS="-* ~arm64"
+RDEPEND="
+	sys-apps/systemd
+"
+DEPEND="${RDEPEND}"
+
+pkg_pretend() {
```

Comment:
Needs to die if not a systemd system

Reply:
Added a message that if the user does not want to use systemd, they are on their own.